## 03_fix-man-page.dpatch by Arjan Oosting <arjan@debian.org>

--- a/doc/man1/c2hs.1
+++ b/doc/man1/c2hs.1
@@ -106,7 +106,7 @@ files and will run
 automatically, passing the appropriate flags.
 
 When used directly, 
-.Bc2hs
+.B c2hs
 is usually called as:
 
 .B c2hs
